playbooks/roles/csi-driver-config/tasks/install.yaml

```diff
@@ -0,0 +1,31 @@
+---
+# tasks file for csi driver installation
+
+- name: Setup csi driver dir
+  file:
+    path: "{{ csidir }}"
+    state: "{{ item }}"
+    mode: '0755'
+  with_items:
+  - directory
+
+- name: Downloading secret template file
+  get_url:
+    url: "{{ secret_template_url }}"
+    dest: "{{ csidir }}/secret.yaml"
+    mode: 0644
+
+- name: Update secret template file
+  shell: |
+    sed -i 's/""/"{{ ibmcloud_api_key }}"/' secret.yaml
+  args:
+    chdir: "{{ csidir }}"
+    warn: false
+
+- name: Create secret resource
+  k8s:
+    state: present
+    src: "{{ csidir }}/secret.yaml"
+
+- name: Create csi driver resource
+  shell: kubectl apply -k "https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/deploy/kubernetes/overlays/stable"
```

Comment:
Hope this URL is correct. Please confirm once.

Reply:
Even if the link is valid, we shouldn't be using the file from main branch.

@Madhan-SWE how other folks are installing the driver? I'm not really comfortable this way for sure!

Also somewhere we need to have better control over version we use for deploying the CSI driver.